Source/Scene/Scene.js

```diff
@@ -2793,16 +2793,22 @@ define([
      * }, Cesium.ScreenSpaceEventType.MOUSE_MOVE);
      *
      * @param {Cartesian2} windowPosition Window coordinates to perform picking on.
+     * @param {Number} [width=3] width of the pick rectangle
+     * @param {Number} [height=3] height of the pick rectangle
      * @returns {Object} Object containing the picked primitive.
      *
      * @exception {DeveloperError} windowPosition is undefined.
      */
-    Scene.prototype.pick = function(windowPosition) {
+    Scene.prototype.pick = function(windowPosition, width, height) {
         //>>includeStart('debug', pragmas.debug);
         if(!defined(windowPosition)) {
             throw new DeveloperError('windowPosition is undefined.');
         }
         //>>includeEnd('debug');
+        // override the rectangle dimensions if defined
+        rectangleWidth = defaultValue(width, 3.0);
+        rectangleHeight = defaultValue(height, rectangleWidth);
+        scratchRectangle = new BoundingRectangle(0.0, 0.0, rectangleWidth, rectangleHeight);
```

Comment:
Check out my latest comment here: https://github.com/AnalyticalGraphicsInc/cesium/pull/5602#discussion_r126710132. I think we can get away without allocating the scratch rectangle in the function.